src/flynt/utils.py

```diff
@@ -0,0 +1,51 @@
+import ast
+from typing import Optional
+
+import astor
+
+from flynt.exceptions import FlyntException
+from flynt.format import set_quote_type, QuoteTypes
+from flynt.linting.fstr_lint import FstrInliner
+
+
+def is_str_literal(node):
+    """Returns True if a node is a string literal. f-string is also a string literal."""
+    return isinstance(node, (ast.Str, ast.JoinedStr))
+
+
+def ast_formatted_value(
+    val: ast.AST,
+    fmt_str: str = None,
+    conversion: Optional[str] = None,
+) -> ast.FormattedValue:
+    if isinstance(val, ast.FormattedValue):
+        return val
+
+    if astor.to_source(val)[0] == "{":
+        raise FlyntException(
+            "values starting with '{' are better left not transformed."
+        )
+
+    if fmt_str:
+        format_spec = ast.JoinedStr([ast_string_node(fmt_str)])
+    else:
+        format_spec = None
+
+    conversion = -1 if conversion is None else ord(conversion.replace("!", ""))
+    return ast.FormattedValue(value=val, conversion=conversion, format_spec=format_spec)
+
+
+def ast_string_node(string: str) -> ast.Str:
+    return ast.Str(s=string)
+
+
+def fixup_transformed(tree: ast.AST) -> str:
```

Comment:
I don't understand the name of this function, why it's "fixup"? Can you please pick a more specific name and add a docstring

Reply:
This was extracted from https://github.com/ikamensh/flynt/blob/071dfb012adca6c13a311bc2f393b8a0907e9cc5/src/flynt/string_concat/transformer.py#L100-L106 because the same thing needs to be done with the static join transformer. "Fixup" seemed apt enough since it does some minor... well, fixups. 😅

Reply:
so maybe we can put in english what it's intention is: 1) remove trailing newline if present, 2) remove double escapes for newline that are artifact of python string representation and 3) replace triple quotes with double quotes (why? I don't remember). Not a blocker, can be added as a follow-up.